src/platform/backends/qemu/CMakeLists.txt

```diff
@@ -24,3 +24,7 @@ target_link_libraries(qemu_backend
   ip_address_pool
   ssh
   yaml)
+
+install(PROGRAMS check_kvm_support
+  DESTINATION bin
```

Comment:
Best use ${CMAKE_RUNTIME_OUTPUT_DIRECTORY} here.

Reply:
Not sure I understand why that is necessary...what is the advantage of doing that?

Reply:
Hmm I suppose there is none, wonder what it gives us to set it in the first place, if we're `install()`ing it manually into bin in the first place.

Reply:
I thought it was needed for 'make install' when building manually.